plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -2659,14 +2663,27 @@ public void registerCoreMembers() {
         // @Group world
         //
         // @Description
-        // TODO: Document Command Details
+        // Specifying a player or list of players will only play

```

Comment:
This is the end of a description, the random facts portion, and not a description in itself.